Content.Client/GameObjects/Components/StackComponent.cs

```diff
@@ -23,12 +27,30 @@ public override int Count
             get => base.Count;
             set
             {
+                var valueChanged = value != Count;
                 base.Count = value;
 
+                if (!Owner.Deleted && valueChanged)
```

Comment:
Does the appearancecomp fail if it's deleted?

Reply:
Hmm. It was possible to be deleted and consumed. E.g. I had an exception thrown when I had 1 element on the ground and an element missing one in hand.

That's why I added the guard. it seems it's no longer necessary.